crates/teloxide/src/sugar.rs

```diff
@@ -0,0 +1,4 @@
+//! Some non-detrimental, but nice additions
```

Comment:
"non-detrimental" is kind of strange in this phrase, maybe just paraphrase it to "Some syntax sugar support for TBA functionality."